python/ray/WebUI.ipynb

```diff
@@ -80,6 +129,278 @@
     "\n",
     "task_search.on_submit(handle_submit)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Task Timeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "path_input = widgets.Button(description=\"View task timeline\")\n",
```

Comment:
When I clicked the "View task timeline" button, I got the following error.

```python
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-6-7de4c7224c71> in handle_submit(sender)
     14     ray.global_state.dump_catapult_trace(tmp)
     15     print("Converting chrome trace to " + tmp2)
---> 16     trace2html = find_trace2html()
     17     subprocess.check_output(["python2", trace2html, tmp, '--output', tmp2])
     18     print("Opening html file in browser...")

<ipython-input-6-7de4c7224c71> in find_trace2html()
      4 def find_trace2html():
      5     trace2html = os.path.join(os.path.expanduser("~"), "catapult", "tracing", "bin", "trace2html")
----> 6     assert os.path.exists(trace2html), "Could not find catapult, please clone it into your home directory from https://github.com/catapult-project/catapult/tree/master/tracing"
      7     return trace2html
      8 

AssertionError: Could not find catapult, please clone it into your home directory from https://github.com/catapult-project/catapult/tree/master/tracing
```

Why do we need to clone this repo?